packages/eas-cli/src/build/metadata.ts

```diff
@@ -64,6 +64,8 @@ export async function collectMetadataAsync<T extends Platform>(
     buildMode: ctx.buildProfile.config ? BuildMode.CUSTOM : BuildMode.BUILD,
     customWorkflowName: ctx.customBuildConfigMetadata?.workflowName,
     developmentClient: ctx.developmentClient,
+    // @ts-expect-error TODO needs @expo/eas-build-job to be relesed
+    requiredPackageManager: ctx.requiredPackageManager,
```

Comment:
I think this type will require it to be ~ `string | undefined`, but the type of it is `string | null`

Reply:
Good spot, I'll add `?? undefined` to match the others!